package.json

```diff
@@ -7,16 +7,22 @@
     "@emotion/styled": "^11.10.4",
     "@mui/icons-material": "^5.10.9",
     "@mui/material": "^5.10.10",
+    "@reduxjs/toolkit": "^1.8.6",
```

Comment:
- [ ] Your frontend proyect has no tests. Kindly add at least some tests to ensure your app's integrity. Please, remember also to include instructions about how to run the tests on your readme.

In the industry, we write tests every day for each feature implemented. You will need this skill for your next brand-new job 💪

Reply:
@Meltrust We have checked all the project requirements and we haven't been able to see testing as a requirement for the frontend. Please let us know where these tests are required.

Reply:
@Daf98 as mentioned in the review summary:

![image](https://user-images.githubusercontent.com/57421823/199534611-9fc49fb9-c6a6-4d72-8105-0462d6c5bd04.png)

Therefore, in this particular project and regarding this review cycle, nothing is a hard requirement. 

This particular special review cycle is meant to prepare you for your final presentation, according to real-world industry standards.